sync_check_terraform/service/Dockerfile-tester

```diff
@@ -21,3 +21,6 @@ WORKDIR /chainsafe
 COPY Gemfile Gemfile.lock ./
 
 RUN gem install bundler && bundle install
+
+COPY ruby_common ruby_common
+COPY * ./
```

Comment:
Are you sure you want to copy over everything? If so, the first line seems redundant, no?

Reply:
fixed using `COPY . .` which copies all files and dir recursively maintaining the folder structure

Reply:
But do we need everything? Why does the tester need, e.g., `docker-compose.yml`?

Reply:
no we don't, changed to copy only required files